contracts/src/v0.8/ccip/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@chainlink/contracts-ccip",
+  "version": "1.5.0",
+  "description": "Chainlink smart contracts for CCIP",
+  "author": "Chainlink devs",
+  "license": "BUSL-1.1",
+  "private": false,
+  "scripts": {
+    "test": "hardhat test --parallel",
+    "size": "hardhat size-contracts",
+    "clean": "hardhat clean",
+    "compile:native": "./../../../scripts/native_solc_compile_all",
+    "compile": "hardhat compile",
+    "coverage": "hardhat coverage",
+    "prepublishOnly": "pnpm compile && ./../../../scripts/prepublish_generate_abi_folder",
+    "publish-beta": "pnpm publish --tag beta",
+    "publish-prod": "pnpm publish --tag latest",
+    "solhint": "solhint --config ./../../../.solhint.json --max-warnings 0 \"./**/*.sol\"",
+    "solhint-test": "solhint --config \"./../../../.solhint-test.json\"  --ignore-path \"./../../../.solhintignore-test\" --max-warnings 0 \"./**/*.sol\""
+  },
+  "files": [
+    "LICENSE.md",
+    "*.sol",
+    "**/*.sol",
+    "abi/v0.8"
+  ],
+  "pnpm": {
+    "_comment": "See https://github.com/ethers-io/ethers.js/discussions/2849#discussioncomment-2696454",
+    "_comment2_logger": "See https://github.com/ethers-io/ethers.js/issues/379 we pin this version since that's what was used in the old yarn lockfile",
+    "overrides": {
+      "@ethersproject/logger": "5.0.6"
+    }
+  },
+  "engines": {
+    "node": ">=18",
+    "pnpm": ">=9"
+  },
+  "devDependencies": {
```

Comment:
TODO: run depcheck or another tool to whittle these down to what's actually needed.